standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/conf/MetastoreConf.java

```diff
@@ -1152,7 +1152,7 @@ public enum ConfVars {
     // Partition management task params
     PARTITION_MANAGEMENT_TASK_FREQUENCY("metastore.partition.management.task.frequency",
       "metastore.partition.management.task.frequency",
-      300, TimeUnit.SECONDS, "Frequency at which timer task runs to do automatic partition management for tables\n" +
+      21600, TimeUnit.SECONDS, "Frequency at which timer task runs to do automatic partition management for tables\n" +
```

Comment:
@rtrivedi12  we can also specify the time in hours instead of seconds.
Can we change this to 6, TimeUnit.HOURS instead?

Reply:
@nrg4878 @saihemanth-cloudera amended! Thanks for the review.